shop/src/main/java/ro/msg/learning/shop/model/Address.java

```diff
@@ -0,0 +1,15 @@
+package ro.msg.learning.shop.model;
+
+import lombok.*;
+
+import javax.persistence.Embeddable;
+import java.io.Serializable;
+
+@Embeddable
+@AllArgsConstructor @NoArgsConstructor @Getter @Setter @ToString

```

Comment:
For "readability" purpose, please arrange these as a stack instead of a long line